lib/bowling.rb

```diff
@@ -0,0 +1,44 @@
+class BowlingGame
+
+  def initialize
+    @rolls = []
+  end
+
+  def rolls(pins)
+    @rolls.push(pins)
+  end
+    
+  def score
+    totalScore = 0
+    rollTurn = 0
```

Comment:
Just a suggestion: a bit of repetition here; when passing rollTurn as an argument for each function, could it be replaced by the index of your @rolls?